source/ui/CLS/CLSValueEditor.cpp

```diff
@@ -55,6 +55,11 @@ CLSValueEditor::~CLSValueEditor()
 
 }
 
+void CLSValueEditor::addLayout(QLayout *newLayout)
```

Comment:
nice catch. I forgot why I added this, but looks like it is never used. I will remove it